cnd/src/btsieve/bitcoin/mod.rs

```diff
@@ -5,10 +5,8 @@ pub use self::{
     bitcoind_connector::{chain_info, BitcoindConnector, ChainInfo},
     cache::Cache,
 };
-use crate::{
-    btsieve::{
-        find_relevant_blocks, BlockByHash, BlockHash, LatestBlock, Predates, PreviousBlockHash,
-    },
+use crate::btsieve::{
```

Comment:
Please wait for https://github.com/comit-network/comit-rs/pull/2218 to be in before merging this and reduce the noise of the change!